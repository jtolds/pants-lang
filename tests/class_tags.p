println ({}.class ==. Dictionary)
println ([].class ==. Dictionary)
println (DynamicVar().class ==. Dictionary)
println ({}.class ==. Array)
println ([].class ==. Array)
println (DynamicVar().class ==. Array)
println ({}.class ==. DynamicVar)
println ([].class ==. DynamicVar)
println (DynamicVar().class ==. DynamicVar)
