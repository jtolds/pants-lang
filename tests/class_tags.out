true
false
false
false
true
false
false
false
true

return code: 0
